File: rps.py
import random

def play_rps(choice):
    # 0 is rock
    # 1 is paper
    # 2 is scissors

    # calc computer choice
    computer_choice = random.randint(0,2)
    # compare computer choice vs user choice
    # determine win, lose or draw
    if computer_choice == choice:
        return "Draw"
    if computer_choice == 0:
        if choice == 1:
            return "You Win"
        elif choice == 2:
            return "You Lose"
    if computer_choice == 1:
        if choice == 0:
            return "You Lose"
        elif choice == 2:
            return "You Win"
    if computer_choice == 2:
        if choice == 0:
            return "You Win"
        elif choice == 1:
            return "You Lose"
    # return result


print(play_rps(2))

File: rps.c
#include <stdio.h>
#include <time.h>
#include <stdlib.h>

void rps(int choice){
    // 1 is rock, 2 is paper, 3 is scissors

    // random is computer's choice
    int random = 1;

    // generate random number from 1 - 3
    srand(time(0));
    random = rand() % 4;

    // determine win, lose, draw and print result
    if (random == 1){
        if (choice == 2) {
            printf("You Win");
        }
        else if (choice == 3) {
            printf("You Lose");
        }
        else {
            printf("Draw");
        }
    }
    if (random == 2){
        if (choice == 3) {
            printf("You Win");
        }
        else if (choice == 1) {
            printf("You Lose");
        }
        else {
            printf("Draw");
        }
    }
    if (random == 3){
        if (choice == 1) {
            printf("You Win");
        }
        else if (choice == 2) {
            printf("You Lose");
        }
        else {
            printf("Draw");
        }
    }
}

int main(){
    // must pass in 1 - 3
    rps(1);
}